meilisearch-release-changelog/v1.12.0.md

```diff
@@ -0,0 +1,137 @@
+# v1.12.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+> [!WARNING]
+> Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.12 introduces significant indexing speed improvements, almost halving the time required to index large datasets. This release also introduces new settings to customize and potentially further increase indexing speed.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Improve indexing speed
+
+Indexing time for huge dataset import (multiple millions of documents) is divided by two!
+
+TBD: give more data and metrics about improvements
```

Comment:
The new default value for the max-indexing-memory parameter is no longer 2/3 of the available memory but 5%. You can find more information on [the related PR](https://github.com/meilisearch/meilisearch/pull/5125).

Reply:
Note, this might change

Reply:
I will let you update the changelog next week or next-next week

Reply:
No longer true: https://github.com/meilisearch/meilisearch/pull/5141